css/styles.css

```diff
@@ -0,0 +1,1010 @@
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.past-summits {
+  width: 90%;
+  margin: auto;
+}
+
+.past-summits-title {
+  font-family: var(--fontLato);
+}
+
+.past-summits-info {
+  font-family: var(--fontLato);
+  width: 80%;
+  margin: auto;
+}
+
+.partners {
+  background-color: #414246;
+  color: #fff;
+  padding: 0.5em;
+}
+
+.summit {
+  position: relative;
+  height: 200px;
+  color: #fff;
+  margin: 2em auto 2em auto;
+  width: 90%;
+}
+
+.summit-one {
+  background: url(../img/assembly-1.png) no-repeat;
+  background-size: cover;
+  background-color: rgba(236, 82, 66, 1);
+  background-blend-mode: multiply;
+  mix-blend-mode: multiply;
+}
+
+.summit-one:hover {
+  background: #272a31;
+  background-size: cover;
+  background-blend-mode: multiply;
+  mix-blend-mode: multiply;
+}
+
+.summit-two {
+  background: url(../img/assembly-1.png) no-repeat;
+  background-size: cover;
+  background-color: rgba(236, 82, 66, 1);
+  background-blend-mode: multiply;
+  mix-blend-mode: multiply;
+}
+
+.summit-two:hover {
+  background: #272a31;
+  background-size: cover;
+  background-blend-mode: multiply;
+  mix-blend-mode: multiply;
+}
+
+.summit-year {
+  margin-bottom: 0;
+  font-size: 2em;
+  font-family: var(--fontLato);
+}
+
+.summit-location {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  flex-wrap: wrap;
+  text-align: center;
+  font-size: 1em;
+  font-family: var(--fontLato);
+}
+
+.sponsors {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 1em;
+  margin: 1em auto 1em auto;
+}
+
+.partners-title {
+  font-family: var(--fontLato);
+  margin-bottom: 8px;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 2em;
+  flex-basis: 100%;
+  padding: 2em;
+}
+
+.home-footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2em;
+  flex-basis: 100%;
+  padding: 2em;
+}
+
+.home-footer-2 {
+  display: none;
+  width: 100%;
+  padding: 0;
+}
+
+.home-partners {
+  background-color: #424146;
+  width: 100%;
+  padding: 0.8em;
+  margin-bottom: 2em;
+}
+
+.home-footer-logo {
+  justify-content: center;
+}
+
+.footer-about-logo {
+  filter: grayscale(1);
+  height: 3em;
+}
+
+.footer-about {
+  justify-content: space-between;
+}
+
+.copyright-text {
+  font-size: 12px;
+  font-family: var(--fontLato);
+}
+
+/* Mobile menu list */
+.close-btn {
+  cursor: pointer;
+}
+
+.close-btn:hover {
+  scale: 1.2;
+}
+
+.menu-overlay {
+  display: none;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  border-radius: 15px;
+  opacity: 1;
+  backdrop-filter: blur(7px);
+  mix-blend-mode: multiply;
+}
+
+.menu-list-mobile {
+  position: fixed;
+  display: none;
+  background: var(--color2);
+  opacity: 1;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
```

Comment:
- [ ] It appears that the images including the logo in the content area are overlapping with the mobile navigation menu. Consider fixing this small issue.

![about section mobile](https://user-images.githubusercontent.com/52143815/191494591-b0ab8bf4-8662-4353-b421-bea84107b596.gif)